common/types/timestamp.go

```diff
@@ -210,7 +210,7 @@ func timestampGetFullYear(t time.Time) ref.Val {
 	return Int(t.Year())
 }
 func timestampGetMonth(t time.Time) ref.Val {
-	return Int(t.Month())
+	return Int(t.Month() - 1)
```

Comment:
Given that our standard library is 1-based for months, but CEL is defined as 0-based, can you comment on the function?

Reply:
Done.